index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="responsive.css">
+    
+    <title>Document</title>
+</head>
+<body>
+  <div id="main">
+      <div id="title">
+          <h1>Nelson Mandela</h1>
+          <h5>The immortal leader of South Africa</h5>
+      </div>
+      <div id="img-div">
+          <img id="image" src="nelsonmandela.jpeg" alt="">
+          <p id="img-caption">The greatest photo taken of the greatest indeed!</p>
+      </div>
+      <div id="tribute-info">

```

Comment:
Please change the spaces on the indentation from 4 to 2 spaces. It is good to avoid using the tab key or set it up to use 2 spaces for readability purposes. Please ensure that is handled.